examples/hub-values.yaml

```diff
@@ -0,0 +1,56 @@
+dash:
+  image:
+    tag: "0.5.58"
+  resources: null
+  # url is configured per workspace.
+  url: "https://dash.test/"
+
+etcd:
+  resources:
+    requests:
+      cpu: "1"
+      memory: 2G
+    limits: null
+  service:
+    annotations:
+      prometheus.io/port: "2379"
+      prometheus.io/scrape: "true"
+  storageClass: "ssd-storage-class"
```

Comment:
By setting this here, no storage class object gets created.

Reply:
Yup, in Hub that is performed by a different step.